src/WhiteLibrary/keywords/screenshot.py

```diff
@@ -1,20 +1,34 @@
 import os
+import robot
 from robot.api import logger
 from robot.libraries.BuiltIn import BuiltIn, RobotNotRunningError
 from robot.utils import get_link_path, is_truthy
 from WhiteLibrary.keywords.librarycomponent import LibraryComponent
 from WhiteLibrary.keywords.robotlibcore import keyword, PY2
-
 from System.Drawing import Bitmap   # noqa: F401 #pylint: disable=unused-import
 from System.Drawing.Imaging import ImageFormat
 from TestStack.White import Desktop
 
 
 class ScreenshotKeywords(LibraryComponent):
-
-    def __init__(self, state):
+    def __init__(self, state, directory):
         super(ScreenshotKeywords, self).__init__(state)
         self.state.screenshooter = self
+        self._screenshot_directory = directory
+
+    @keyword
+    def set_screenshot_directory(self, path):
+        """Sets the directory where WhiteLibrary stores screenshots and returns the previously set path.
+
+        If the given directory does not exist in the system, it will be created.
```

Comment:
Maybe point out that the directory will be created only when the first screenshot is taken?